toolchain/lower/testdata/interop/cpp/return.carbon

```diff
@@ -98,6 +98,20 @@ fn Var() {
   var x: Cpp.X = Cpp.Make();
 }
 
+// --- indirect_return_with_args.carbon
```

Comment:
Out of curiosity, does this test fail without the fix?

Reply:
Yes, it asserts in `-c dbg` because we generate a wrong declaration of the thunk due to the incomplete return type, then try to build a malformed call to it.